pkg/services/sqlstore/apikey_test.go

```diff
@@ -66,7 +66,7 @@ func TestApiKeyDataAccess(t *testing.T) {
 			//expires in one day
 			cmd := m.AddApiKeyCommand{OrgId: 1, Name: "key-with-negative-lifespan", Key: "asd3", SecondsToLive: -3600}
 			err := AddApiKey(&cmd)
-			assert.EqualError(t, err, "Negative value for SecondsToLive")
+			assert.EqualError(t, err, m.ErrInvalidApiKeyExpiration.Error())
```

Comment:
Should be `models.ErrInvalidApiKeyExpiration` so the import at the top shouldn't be aliased. Think we decided that, but can't find any documentation about it.